catsoop/lti.py

```diff
@@ -0,0 +1,297 @@
+# This file is part of CAT-SOOP
+
+"""
+LTI Tool Provider interface
+"""
+
+import os
+import uuid
+import string
+import random
+import hashlib
+import pylti.common
```

Comment:
I'm going to go ahead and add `pylti` to the pip `requirements.txt` as well, when I merge this in.

Reply:
Oh woops, didn't notice it was already there.